index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE HTML>
+<html ng-app="Capture">
+  <head>
+    <title>Capture</title>
+    <link href='http://fonts.googleapis.com/css?family=Poiret+One' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css">
+    <link rel="stylesheet" href="assets/css/app.css">
+    <script src="bower_components/jquery/dist/jquery.js"></script>
+    <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/angular-resource/angular-resource.js"></script>
+    <script src="app/app.module.js"></script>
+    <script src="app/components/captureTasksController.js"></script>
+  </head>
+
+  <body ng-controller="CaptureTasksController as taskCtrl">
+
+    <header class="row">
+      <h1 class="col-md-6 col-md-offset-3 header">Capture</h1>
+      <h2 class="col-md-6 col-md-offset-3 header header__lower">The repository to allow you to GTD</h2>
+    </header>
+
+    <userinput class="row">
+      <form class="col-md-6 col-md-offset-3">
+        <input id="task" type="text" placeholder="Offload ..." ng-model="taskCtrl.loadTask">
+        <button class="form__button--submit btn btn-primary" ng-click="taskCtrl.addTask(taskCtrl.loadTask)">Submit</button>
+      </form>
+    </userinput>
```

Comment:
If you wanted to be super clear with what it does, you have two main options:

1. CSS classes: `<article class="user-input">` (which you can BEM too)
2. ARIA tagging: `<article role="user-input">`